Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.ContainInConsecutiveOrder.cs

```diff
@@ -0,0 +1,293 @@
+﻿using System;
+using FluentAssertions.Execution;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Collections;
+
+/// <content>
+/// The [Not]ContainInOrder specs.
+/// </content>
+public partial class CollectionAssertionSpecs
+{
+    public class ContainInExplicitOrder
```

Comment:
explicit -> consecutive.
here and more places in the PR.

Reply:
Sorry, I thought I caught them all. I'll double check